tests/__init__.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+import tempfile
+import clkhash
+
+TESTDATA = os.path.join(
+    os.path.dirname(__file__),
+    'testdata'
+)
+
+SIMPLE_SCHEMA_PATH = os.path.join(TESTDATA, 'simple-schema.json')
+
+SAMPLE_DATA_SCHEMA_PATH = os.path.join(TESTDATA, 'dirty-data-schema.json')
```

Comment:
not used anywhere

Reply:
This is used when testing good schema

Reply:
SAMPLE_DATA_SCHEMA_PATH is not used anywhere